.github/workflows/full-help-docs.yml

```diff
@@ -0,0 +1,26 @@
+name: CLI Help Doc
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  pull-requests: read
+
+jobs:
+  doc_check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Generate help doc
+      run: |
+        echo "xtask=$(cargo xtask markdown && git status -s docs/soroban-cli-full-docs.md)" >> $GITHUB_OUTPUT
+      id: run_generate
+
+    - name: Check diff
+      if: ${{ steps.run_generate.outputs.xtask }} != ""
+      uses: actions/github-script@v3
+      with:
+        script: |
+            core.setFailed('You need to run `cargo xtask markdown` to update the full help docs. You can also install the pre-commit hook as explained in the README to do this automatically on every commit.')
```

Comment:
```suggestion
            core.setFailed('You need to run `cargo xtask markdown` to update the full help docs. You can also install the pre-commit hook as explained in the README to do this automatically on every commit, or run: cargo xtask markdown')
```

Reply:
This made it a bit redundant 😄

Reply:
I more-or-less changed this back; I hope you don't mind. Having the `cargo xtask markdown` in there twice seems unnecessary to me.